src/background/background-process.js

```diff
@@ -277,18 +344,16 @@ export default class BackgroundProcess {
    */
   async navigationCompletedListener (details) {
     let finalWindowId = this.defineCurrentWindowIdForActivation(details)
-
     let tmpTabUniqueId = Tab.createUniqueId(details.tabId, finalWindowId)
 
     if (this.tabs.has(tmpTabUniqueId)) {
+      let tab = this.tabs.get(tmpTabUniqueId)
       // make sure this is a tab we know about
       if (details.frameId === 0) {
         // AND that it's not an iframe event
-        // If content script was loaded to that tab, restore it to the state it had before
-        let tab = this.tabs.get(tmpTabUniqueId)
-        tab.restore()
         try {
           await this.loadContentData(tab)
+          this.setDefaultTabState(tab)
```

Comment:
I think we want to remove this because we want to retain panel/tab state when a page is reloaded or navigated to a new page

Reply:
That's correct, thanks for fixing this. I was just under the false impression that we do need to reset it.